spec/ttml1.xml

```diff
@@ -7781,6 +7781,9 @@ interval; i.e., the interval is left-wise closed.</p>
 <p>The semantics of the <att>begin</att> attribute are those defined
 by <bibref ref="smil21"/>, &sect; 10.4.1,
 while taking into account any overriding semantics defined by this specification.</p>
+<note role="clarification">
```

Comment:
Replace with following "See ``<bibref ref="smil21"/>``, &sect; 10.4.1, ``<emph>``Begin value semantics``</emph>``, for information about default timing."

Reply:
I'm happy with either the pull request version or @skynavga 's proposed alternative.

Also as per https://github.com/w3c/ttml1/issues/224#issuecomment-268787562 it would be useful to state that TTML does not define any epoch or epoch offset, and that if those are relevant they need to part of the document processing context.

Reply:
> Also as per #224 (comment) it would be useful to state that TTML does not define any epoch or epoch offset, and that if those are relevant they need to part of the document processing context.

Please file a separate issue and propose specific text.

Reply:
@skynavga The original comment was behavior when `begin` is not specified, so I think the note needs to mention that. What about:

_See `<bibref ref="smil21"/>`, § 10.4.1, `<emph>Begin value semantics</emph>` when no `begin` attribute is specified`_

Reply:
> Please file a separate issue and propose specific text.

#266